internal/providers/terraform/azure/testdata/storage_account_test/storage_account_test.golden

```diff
@@ -1,91 +1,139 @@
+ ├─ Capacity (next 450TB)                                 512,000  GB-months             $10,223.62 
+ ├─ Capacity (over 500TB)                               5,436,800  GB-months            $104,038.60 
+ ├─ Write operations                                          600  10K operations            $30.00 
+ ├─ List and create container operations                      600  10K operations            $30.00 
+ ├─ Read operations                                            60  10K operations             $0.24 
+ ├─ All other operations                                      600  10K operations             $2.40 
+ └─ Blob index                                                 60  10K tags                   $1.80 
+                                                                                                    
+ azurerm_storage_account.bb_Standard_RAGRS_Cool                                                     
+ ├─ Capacity                                            7,000,000  GB-months            $297,500.00 
+ ├─ Write operations                                          700  10K operations           $140.00 
+ ├─ List and create container operations                      700  10K operations            $70.00 
+ ├─ Read operations                                            70  10K operations             $0.70 
+ ├─ All other operations                                      700  10K operations             $2.80 
+ ├─ Data retrieval                                          7,000  GB                        $70.00 
+ ├─ Data write                                              7,000  GB                        $35.00 
+ └─ Blob index                                                 70  10K tags                   $4.83 
+                                                                                                    
+ azurerm_storage_account.bb_Standard_RAGRS_Hot                                                      
+ ├─ Capacity (first 50TB)                                  51,200  GB-months              $3,015.68 
+ ├─ Capacity (next 450TB)                                 512,000  GB-months             $28,950.53 
+ ├─ Capacity (over 500TB)                               7,436,800  GB-months            $402,985.32 
+ ├─ List and create container operations                      800  10K operations            $80.00 
+ ├─ Read operations                                            80  10K operations             $0.32 
+ ├─ All other operations                                      800  10K operations             $3.20 
+ └─ Blob index                                                 80  10K tags                   $5.52 
+                                                                                                    
+ azurerm_storage_account.bb_without_usage_file                                                      
+ ├─ Capacity                                      Cost depends on usage: $0.04 per GB-months        
```

Comment:
The branch needs a rebase to pick up the "Monthly cost depends..." output changes.